Advice

florcz opened the pull request.

redrambles commented:
Hey @florcz,

This is great!! We have your advice file, your custom css file and a link to your advice file in the main index.html page. Excellent.

The only issue is that you also modified the main css/style.css file - which is the 'fallback' css file that everyone who doesn't provide a custom css page is using, so we are going to need you to restore it to its original state. If you have a look at the diff view of your pull request here: https://github.com/skillcrush/105-practice/pull/1479/files - you can see how the file was changed, and what was there before it was modified. You can just copy/paste the original version of the file contents and place them inside your css/style.css file - save, add and commit and then push up your advice branch once more.

Once that is done, we'll be all set to merge in your code. Let us know if you have any questions!

Cheers,
Ann :)

florcz commented:
Hi Ann,

I'm pretty sure I fixed it ;D... I followed all the instructions you gave
me, do I need to submit a new pull request?

Thanks for your help!
Flor

On Fri, Aug 14, 2020 at 2:34 PM Ann Cascarano <notifications@github.com>
wrote:

> Hey @florcz <https://github.com/florcz>,
>
> This is great!! We have your advice file, your custom css file and a link
> to your advice file in the main index.html page. Excellent.
>
> The only issue is that you also modified the main css/style.css file -
> which is the 'fallback' css file that everyone who doesn't provide a custom
> css page is using, so we are going to need you to restore it to its
> original state. If you have a look at the diff view of your pull request
> here: https://github.com/skillcrush/105-practice/pull/1479/files - you
> can see how the file was changed, and what was there before it was
> modified. You can just copy/paste the original version of the file contents
> and place them inside your css/style.css file - save, add and commit and
> then push up your advice branch once more.
>
> Once that is done, we'll be all set to merge in your code. Let us know if
> you have any questions!
>
> Cheers,
> Ann :)
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/skillcrush/105-practice/pull/1479#issuecomment-674208463>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AQPY3J5A36P5ZRLUN5CJ5SDSAV7TZANCNFSM4P7XBHZQ>
> .
>

acodeinprogress commented:
Hey @florcz,

Great job! Your code is all [merged in](https://media.tenor.com/images/4a6158ad4f30c71f4fecf224cec12eb1/tenor.gif).


xoxo,
Sharon
